client/gulpfile.js

```diff
@@ -155,6 +146,18 @@ const watch = () => {
   });
 };
 
+const watch = () => {
+  gulp.watch(
+    "src/app/**/*.{ts,tsx,css}",
+    { ignoreInitial: false },
+    javascriptWatch
+  );
+  gulp.watch("src/worker/**/*.{js,ts,tsx}", { ignoreInitial: false }, worker);
+  gulp.watch("src/public/**/*", { ignoreInitial: false }, public);
+  gulp.watch("vendor/wormhole-william/**/*.go", { ignoreInitial: false }, wasm);
+  gulp.series(start);
```

Comment:
This doesn't seem to run the server, but we can call the function directly
```suggestion
  start();
```